vispy/scene/cameras/base_camera.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, interactive=True, flip=None, up='+z', parent=None,
 
         # We initialize these parameters here, because we want these props
         # available in all cameras. Note that PanZoom does not use _center
-        self._fov = 0.0
+        self._fov = self.fov(kwargs.pop('fov', 0.0))
```

Comment:
Is this what you meant?

```suggestion
        self._fov = kwargs.pop('fov', 0.0)
```

Reply:
no, because this would allow a value beyond the 0-180° range to be set.

please note that a change I made in the fov setter allows a 180° value to be set (made it out of conveniency) but it might trigger a singularity... cannot test just now, busy hacking a more ergonomic mouse/keyboard setup

Reply:
Ok but the code you have here is invalid. `.fov` is a property not a method. That's why the tests are failing. _fov has to be initialized to some value. The subclasses are doing that already. There is technically no need to set this here...just kind of a weird design.

Reply:
There isn't even a `kwargs` here, or am I confused? This is just initalization, so it should be left as `self._fov = 0.0`; the setter will be called regardless by subclasses.

Reply:
:confused: I could have sworn there was a `**kwargs` in this init method. Now I'm very confused. Maybe this line should just go back to the way it was. There is no `fov` parameter to the BaseCamera, it is only in the perspective.py base classes and those already set `self.fov`. This line with `self._fov` needs to be here so the instance variable exists (mostly for flake8/linting checks) on init.